backend/src/test/java/com/huellapositiva/integration/ManageNewsletterExcelActionShould.java

```diff
@@ -0,0 +1,72 @@
+package com.huellapositiva.integration;
+
+import com.huellapositiva.domain.actions.ManageNewsletterExcelAction;
+import com.huellapositiva.domain.model.valueobjects.EmailAddress;
+import com.huellapositiva.domain.service.EmailCommunicationService;
+import com.huellapositiva.domain.service.RemoteStorageService;
+import com.huellapositiva.infrastructure.orm.repository.JpaVolunteerRepository;
+import com.huellapositiva.util.TestData;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.context.annotation.Import;
+
+import java.io.IOException;
+
+import static com.huellapositiva.util.TestData.DEFAULT_EMAIL;
+import static com.huellapositiva.util.TestData.DEFAULT_PASSWORD;
+import static org.mockito.Mockito.verify;
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@ExtendWith(MockitoExtension.class)
```

Comment:
Redundant? SpringBootTest annotation probably does that already, can verify though

Reply:
Yes, I removed the extendwith annotation